libs/SalesforceSDKCore/Images.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
In the shared/resoources/ we do have an Image asset catalog. I can add this as a separate asset catalog in there.